admin-src/index.js

```diff
@@ -21,36 +26,34 @@ const App = () => {
 
   const onUserLogin = () => setUserLoggedIn(true)
 
-  if (userLoggedIn) {
+  if (!userLoggedIn) {
```

Comment:
Rollback this change. Otherwise you are bypassing the login process outside dev environment.